packages/docusaurus-theme-openapi/src/theme/ApiItem/index.tsx

```diff
@@ -8,125 +8,51 @@
-
-  const renderTocDesktop =
-    canRenderTOC && (windowSize === "desktop" || windowSize === "ssr");
-
-  return (
-    <>
-      <Seo {...{ title, description, keywords, image }} />
-      <div className="row">
-        <div className={clsx("col", api ? "col--7" : "col--9")}>
-          <DocVersionBanner />
-          <div className={styles.apiItemContainer}>
-            <article>
-              <DocBreadcrumbs />
-              <DocVersionBadge />
-
-              <div className={clsx("theme-api-markdown", "markdown")}>
-                {canRenderTOC && (
-                  <TOCCollapsible
-                    toc={ApiContent.toc}
-                    minHeadingLevel={tocMinHeadingLevel}
-                    maxHeadingLevel={tocMaxHeadingLevel}
-                    className={clsx(
-                      ThemeClassNames.docs.docTocMobile,
-                      styles.tocMobile
-                    )}
-                  />
-                )}
-                {shouldAddTitle && (
-                  <header>
-                    <Heading as="h1">{title}</Heading>
-                  </header>
-                )}
-                <ApiContent />
-              </div>
-              <DocItemFooter {...props} />
-            </article>
+export default function ApiItem(props: typeof Props): JSX.Element {
+  let api = props.content.frontMatter.api ?? false;
+  if (!api) {
+    return (
+      <>
+        <DocItem {...props} />
+      </>
+    );
+  } else {
+    let next = props.content.metadata.next;
+    let previous = props.content.metadata.previous;
+    props.content.metadata.next = null;
+    props.content.metadata.previous = null;
+    props.content.metadata.toc = null;
+    return (
+      <div className={styles.apiItemContainer}>
+        <div className={clsx("theme-api-markdown", "markdown")}>
+          <div className="row">
+            <div className="col col--7">
+              <DocItem {...props} />
+            </div>
+            <div className="col col--5">
+              <ApiDemoPanel item={api} />
+            </div>
+          </div>
+          <div className="row">
+            <div className={clsx("col")}>
+              <DocPaginator previous={previous} next={next} />
+            </div>
           </div>
-        </div>
-        <div className={clsx("col", api && "col--5")}>
-          {api && (
-            <ApiDemoPanel
-              item={api}
-              showExecuteButton={showExecuteButton}
-              showManualAuthentication={showManualAuthentication}
-            />
-          )}
-          {renderTocDesktop && (
-            <TOC
-              toc={ApiContent.toc}
-              minHeadingLevel={tocMinHeadingLevel}
-              maxHeadingLevel={tocMaxHeadingLevel}
-              className={ThemeClassNames.docs.docTocDesktop}
-            />
-          )}
-        </div>
-      </div>
-      <div className="row">
-        <div className={clsx("col", api ? "col--7" : "col--9")}>
```

Comment:
Full-width pagination will be too wide. Let's go ahead and revert back to stay consistent with how Docusaurus handles paginator width/position by default. The conditional will handle setting the width for api and non-api docs. Thanks!